sympl/_components/netcdf.py

```diff
@@ -81,6 +96,22 @@ def store(self, state):
                 cache_state = {name: state[name] for name in name_list}
             else:
                 cache_state = state.copy()
+
+            # replace cached variable names with their aliases
+            for longname, shortname in self._aliases.items():
+                for full_var_name in cache_state.keys():
+                    # replace any string in the full variable name that matches longname
+                    # example: if longname is "temperature", shortname is "T", and
+                    #    full_var_name is "temperature_tendency_from_radiation", the
+                    #    alias_name for the variable would be: "T_tendency_from_radiation"
```

Comment:
First check if full_var_name is already the empty string, and raise an appropriate error if it is. You should also write a test for this case.